### ISSUE TITLE
Add parameter to "slash run" for forcing/forbidden console colors

### ISSUE THREAD

**Issue by ayalash** [getslash/slash#635]
Logbook supports forbidden/forcing log colors: since https://github.com/getlogbook/logbook/pull/239

slash run should support it too.

**Comment by ayalash** [comment]
As far as I know, there are too conventions for it:
* Example from `grep` man page: 
```
--color[=WHEN], --colour[=WHEN]
    ... WHEN is never, always, or auto.
```
* Example from `ag` man page:
``` 
--[no]color
    Print color codes in results. Enabled by default.
```

**Comment by vmalloc** [comment]
@ayalash I prefer the first convention

**Comment by ayalash** [comment]
@vmalloc You preferred that I use `grep` like convention, but I just remembered that we have `--force-color` and `--no-color` flags to `slash list` and `slash list-config`. Shouldn't we have the same convention for all sub-commands? Should I change `list` and `list-config` to use `--color[=WHEN]` as well?

**Comment by vmalloc** [comment]
Yup. We should

**Comment by ayalash** [comment]
@vmalloc Should I support the "old" flags (for list commands) with deprecation, or should I just change them to the new ones?

**Comment by vmalloc** [comment]
@ayalash I meant to align the new ones according to the old ones, not the other way around. The old ones seem reasonable as well, so no big deal

**Comment by ayalash** [comment]
oh, ok... I'm happy I asked :)

**Comment by ayalash** [comment]
@vmalloc Can you please release a version of Lobook?
This issue should depend on the code you merged on https://github.com/getlogbook/logbook/pull/239.

**Comment by vmalloc** [comment]
Released 1.1.0. Will be on Pypi soon
